src/main/java/com/bcarnevale/minesweeperweb/GameStuff/Reader.java

```diff
@@ -0,0 +1,29 @@
+package com.bcarnevale.minesweeperweb.GameStuff;
+
+import Game.IReader;
+
+public class Reader implements IReader {
```

Comment:
Are the `Reader` and `Writer` classes actually being used, or is it just `WebReader` and `WebWriter` that are being used? If we're not using `Reader` and `Writer` any more, please delete them